sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -96,52 +103,46 @@ enum class aspect {
 }
 ----
 
-If a SYCL device has the `ext_oneapi_bfloat16` aspect, then it natively
-supports conversion of values of `float` type to `bfloat16` and back.
+If a SYCL device has the `ext_oneapi_bfloat16` aspect, then it supports
+`bfloat16` conversions described in the next section.
```

Comment:
Sorry, I thought I made a comment here on Friday, but I don't see it now.  This description of `aspect::ext_oneapi_bfloat16` was added back in 68308576d98aedf5be2c60e30de66c837e614aff.  I don't think we want this.  We say earlier in the spec that "The bfloat16 type is supported on all devices."  This contradicts the statement here that a device only supports `bfloat16` if it has this aspect.

I think our intention is that all devices support `bfloat16`, either natively or through software emulation.  Therefore, we don't need this aspect.  Can we just remove it?

Reply:
We've gone back and forth on this. Another reviewer wanted to keep bfloat16 conversions and bfloat16 math as separate aspects, which makes them visible, and so they have to be documented. If we agree to keep both aspects then I can change the doc. But lets agree on the number of aspects first.

Reply:
I think the other reviewer wanted to name the aspect that gates the math functions in a more obvious way, and the name `ext_oneapi_bfloat16_math_functions` seems reasonable.

At this point, it seems like we don't need the aspect `ext_oneapi_bfloat16`.  Can we remove both the implementation and the documentation for it?  Is anyone advocating that we keep it?

Reply:
Removing ext_oneapi_bfloat16 and replacing with ext_oneapi_bfloat16_math_functions would be a "breaking" ABI change. You're sure you want to do it?

Reply:
Aren't you already breaking both API and ABI with this change?  For example, the `bfloat16` type was moved to a different namespace.

Reply:
Yes, that's true.
Still, you're sure there is no value to keeping the bfloat16 aspect and have it mean native support for bfloat16 conversions?

Reply:
We don't have anyone asking for a query like that, and I'd rather not add one unless there is a clear use case.  Even if we did add an aspect to do this query, we'd want to choose a different name like "ext_oneapi_native_bfloat16".  I'd rather remove `ext_oneapi_bfloat16` now.  We can consider adding `ext_oneapi_native_bfloat16` later if someone asks for it.